modules/imgcodecs/src/ios_conversions.mm

```diff
@@ -116,6 +118,8 @@ void UIImageToMat(const UIImage* image,
         if (!alphaExist)
             bitmapInfo = kCGImageAlphaNoneSkipLast |
                                 kCGBitmapByteOrderDefault;
+        else
+            m = cv::Scalar(0);
```

Comment:
Looks like here we have an alpha channel and going to store it to RGBA image. Shouldn't we use kCGImageAlphaLast to store alpha channel as well?